recipes/minizip/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "minizip" %}
+{% set version = "4.0.1" %}
+{% set sha256 = "63e47a2b4dbac0da501f43f4da74f118dfb3ef0dee0ffbbe89428271002260f8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zlib-ng/minizip-ng/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Minor suggestion but we don't need this jinja variable.

```suggestion

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/zlib-ng/minizip-ng/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
  sha256: 63e47a2b4dbac0da501f43f4da74f118dfb3ef0dee0ffbbe89428271002260f8
```